pkg/utils/utils.go

```diff
@@ -22,6 +22,26 @@ func CheckNodeNameIdentical(s1, s2 string) bool {
 	return strings.Split(s1, ".")[0] == strings.Split(s2, ".")[0]
 }
 
+// GetHostname returns OS's hostname if 'hostnameOverride' is empty; otherwise, return 'hostnameOverride'.
+func GetHostname(hostnameOverride string) (string, error) {
+	hostName := hostnameOverride
+	if len(hostName) == 0 {
+		nodeName, err := os.Hostname()
+		if err != nil {
+			return "", fmt.Errorf("couldn't determine hostname: %v", err)
+		}
+		hostName = nodeName
```

Comment:
why do you need the nodeName?
you can just 
hostName , err := os.Hostname() and remove Line 33

Reply:
That would define a new hostName var in the inner scope (because of the `:=`)

i could do:
```
if len(hostName) == 0 {
  var err
  hostName, err = os.Hostname
....
}
...
```

it would save on temp var.